src/components/common/SwitchTheme.js

```diff
@@ -14,8 +45,16 @@ const SwitchTheme = () => (
           onChange={(e) => toggleTheme(e.target.checked ? 'dark' : 'light')}
           checked={'dark' === theme}
         />
-        <label htmlFor="switch-theme__input" className="switch-theme__label">
-          Switch to {'dark' === theme ? 'light mode' : 'dark mode'}
+        <label
+          htmlFor="switch-theme__input"
+          className="switch-theme__label"
+          title={`Switch to ${'dark' === theme ? 'light' : 'dark'} mode`}
+        >
+          {' '}
```

Comment:
Why?

Reply:
It is an ugly trick to avoid this eslint error : 
https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/label-has-associated-control.md
But I'm going to deactivate the rule in this file instead.